js/chat_app/src/modals/ChatForm/Form.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import Button from '@mui/material/Button';
+import DialogTitle from '@mui/material/DialogTitle';
+import Dialog from '@mui/material/Dialog';
+import DialogContent from '@mui/material/DialogContent';
+import TextField from '@mui/material/TextField';
+import { useDispatch, useSelector } from 'react-redux';
+import CreateChatSelect from './Select';
+import { getAllUsersThunk } from '../../redux/slices/usersSlice';
+import { createChatThunk } from '../../redux/slices/chatSlice';
+
+export default function CreateChatForm({ onClose, open, authToken }) {
+  const [chatName, setChatName] = useState('');
+  const [errorChatName, setErrorChatName] = useState(false);
+  const [resetMembersTrigger, setResetMembersTrigger] = useState(true);
+
+  const dispatch = useDispatch();
+
+  const users = useSelector((state) => state.users.allUsers);
+
+  useEffect(() => {
+    // TODO: check it after implement loginization
+    dispatch(getAllUsersThunk(authToken));
+  }, [authToken]);
+
+  const handleClose = () => {
+    setChatName('');
+    onClose();
+  };
+
+  const handleChatName = (e) => {
+    setChatName(e.target.value);
+  };
+
+  const handleSendButton = () => {
+    const paramsForCreateChat = {
+      chatName,
+      authToken
+    };
+
+    dispatch(createChatThunk(paramsForCreateChat));
```

Comment:
what if the request will fail? i would probably pass callback to the action and call it after everything is done